src/js/structs/Job.js

```diff
@@ -18,11 +19,42 @@ module.exports = class Job extends Item {
     return this.get('id');
   }
 
+  getLastRunStatus() {
+    let {lastFailureAt, lastSuccessAt} = this.get('status');
```

Comment:
Let's also account for undefined statuses (ie a one-off job that has never run).